wrapper.sh

```diff
@@ -133,8 +164,41 @@ if [[ $create == true ]]; then
     git submodule init
     git submodule import
  
-    echo "import local submodules"
-    git submodule import --recursive --remote --merge
+    echo "update local submodules"
+    #git submodule update --recursive --merge
+
+    #remove _locale directory (only translate English lessons)
```

Comment:
Don't worry about what this is doing too much. The issue really is that we can't pull updates for submodules from the right branch as David uses the "master" branch renamed as "gh-pages" so the history doesn't match when you pull "origin".

Merging updates might be tricky because of this but I should be able to sort it out.